no python suffix like ".cpython-36m-x86_64-linux-gnu": provides a consistent
naming across architecture.

Also don't overwrite the "real" cmake file with the generic wrapper's one: they
have the same name if no python suffix.

--- a/sources/shiboken6/cmake/ShibokenSetup.cmake
+++ b/sources/shiboken6/cmake/ShibokenSetup.cmake
@@ -74,6 +74,11 @@ if(NOT PYTHON_CONFIG_SUFFIX)
         "PYTHON_CONFIG_SUFFIX is empty. It should never be empty. Please file a bug report.")
 endif()
 
+# no suffix like '.cpython-36m-x86_64-linux-gnu'
+set(PYTHON_EXTENSION_SUFFIX "")
+set(PYTHON_CONFIG_SUFFIX "")
+set(PYTHON_SHARED_LIBRARY_SUFFIX "")
+
 message(STATUS "PYTHON_EXTENSION_SUFFIX:      ${PYTHON_EXTENSION_SUFFIX}")
 message(STATUS "PYTHON_CONFIG_SUFFIX:         ${PYTHON_CONFIG_SUFFIX}")
 message(STATUS "PYTHON_SHARED_LIBRARY_SUFFIX: ${PYTHON_SHARED_LIBRARY_SUFFIX}")
--- a/sources/shiboken6/data/CMakeLists.txt
+++ b/sources/shiboken6/data/CMakeLists.txt
@@ -24,16 +24,11 @@ configure_package_config_file(
      INSTALL_DESTINATION "${LIB_INSTALL_DIR}/cmake/Shiboken6"
 )
 
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/Shiboken6Config.cmake.in"
-               "${CMAKE_CURRENT_BINARY_DIR}/Shiboken6Config.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/Shiboken6ConfigVersion.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/Shiboken6ConfigVersion.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/shiboken6.pc.in"
                "${CMAKE_CURRENT_BINARY_DIR}/shiboken6${shiboken6_SUFFIX}.pc" @ONLY)
 
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/Shiboken6Config.cmake"
-        DESTINATION "${LIB_INSTALL_DIR}/cmake/Shiboken6")
-
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/install/Shiboken6Config${PYTHON_CONFIG_SUFFIX}.cmake"
         DESTINATION "${LIB_INSTALL_DIR}/cmake/Shiboken6")
 
--- a/sources/pyside6/libpyside/CMakeLists.txt
+++ b/sources/pyside6/libpyside/CMakeLists.txt
@@ -168,8 +168,6 @@ configure_package_config_file(
     PATH_VARS PYSIDE_PYTHONPATH PYSIDE_TYPESYSTEMS PYSIDE_GLUE
 )
 
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/PySide6Config.cmake.in"
-               "${CMAKE_CURRENT_BINARY_DIR}/PySide6Config.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/PySide6ConfigVersion.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/PySide6ConfigVersion.cmake" @ONLY)
 
@@ -194,9 +192,6 @@ install(TARGETS pyside6 EXPORT PySide6Wh
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pyside6${pyside6_SUFFIX}.pc"
         DESTINATION "${LIB_INSTALL_DIR}/pkgconfig")
 
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/PySide6Config.cmake"
-        DESTINATION "${LIB_INSTALL_DIR}/cmake/PySide6")
-
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/install/PySide6Config${SHIBOKEN_PYTHON_CONFIG_SUFFIX}.cmake"
         DESTINATION "${LIB_INSTALL_DIR}/cmake/PySide6")
 
